doc/api/n-api.md

```diff
@@ -3556,6 +3557,26 @@ NAPI_EXTERN napi_status napi_is_promise(napi_env env,
 - `[out] is_promise`: Flag indicating whether `promise` is a native promise
 object - that is, a promise object created by the underlying engine.
 
+## Script execution
+
+N-API allows you to execute a string containing valid JavaScript using the
+underlying JavaScript engine.
+
+### napi_run_script
+<!-- YAML
+added: REPLACEME
+-->
+```C
+NAPI_EXTERN napi_status napi_run_script(napi_env env,
+                                        const char* script,
+                                        napi_value* result);
+```
+
+- `[in] env`: The environment that the API is invoked under.
+- `[in] script`: The string containing the script to execute.
```

Comment:
Make it clear that this is in UTF-8? Or just take a string napi\_value instead?

Reply:
Maybe it'd be easier to take a `napi_value` instead.